apps/slack/src/pages/api/webhooks/order-created.ts

```diff
@@ -71,12 +72,21 @@ export const orderCreatedWebhook = new SaleorAsyncWebhook<OrderCreatedWebhookPay
   event: "ORDER_CREATED",
   apl: saleorApp.apl,
   query: OrderCreatedGraphqlSubscription,
+  isActive: false,
 });
 
+const isValidUrl = (url: string) => {
+  try {
+    new URL(url);
+  } catch (e) {
+    return false;
+  }
+};
```

Comment:
suggestion: I think this could be extracted into a separate utility, since the same snippet is used in [apps/slack/src/pages/api/configuration.ts](https://github.com/saleor/apps/pull/1153/files#diff-1bf5f278ae954bb06c1828584f095dfea07604202a2061dce3d29ad9977b9b12)

Reply:
fixed